deny.toml

```diff
@@ -64,9 +60,6 @@ allow = [
     "Unicode-3.0",
     "Zlib"
 ]
-copyleft = "warn"
-allow-osi-fsf-free = "neither"
```

Comment:
What does removing this mean? Is it also making something stricter?

Reply:
I believe yes, this is more strict now. The tool no longer allows you to `warn`; Something is either an error or it isn't. Currently, we're only going to error on licenses of crates that we directly depend on, and we don't depend on any copyleft, osi/fsf free stuff. If we do start depending on those in the future, we can allowlist them.

Reply:
okay, i agree based on this:
https://github.com/EmbarkStudios/cargo-deny/blob/1c8b8908d9db080ca4e226905c25cf8dff38bc01/docs/src/checks/licenses/cfg.md?plain=1#L59